lib/ansiblelint/rules/YamllintRule.py

```diff
@@ -0,0 +1,94 @@
+import logging
+import os
+import sys
+from typing import TYPE_CHECKING, List
+
+from ansiblelint.rules import AnsibleLintRule
+from ansiblelint.skip_utils import get_rule_skips_from_line
+
+if TYPE_CHECKING:
+    from ansiblelint.errors import MatchError
+
+_logger = logging.getLogger(__name__)
+
+# yamllint is a soft-dependency (not installed by default)
+try:
+    from yamllint.config import YamlLintConfig
+    from yamllint.linter import run as run_yamllint
+except ImportError:
+    pass
+
+
+YAMLLINT_CONFIG = """
+extends: default
+rules:
+  document-start: disable
+  # 160 chars was the default used by old E204 rule, but
+  # you can easily change it or disable in your .yamllint file.
+  line-length:
+    max: 160
+"""
+
+DESCRIPTION = """\
+Rule violations reported by YamlLint when this is installed.
+
+You can fully disable all of them by adding 'YAML' to the 'skip_list'.
```

Comment:
Just to be sure: This is supposed to with `YAML` in the skip_list, not with `yaml`? To me it seems like the other way around?

Reply:
That is an error, correct key is 'yaml', I will fix it.

Reply:
Fixing via https://github.com/ansible-community/ansible-lint/pull/1335